### PR TITLE
Add Immutable X

### PR DESCRIPTION
**Pull request by 0xmichalis**
# [defillama-server PR here](https://github.com/DefiLlama/defillama-server/pull/3667)

##### Name (to be shown on DefiLlama):

Immutable

##### Twitter Link:

https://twitter.com/Immutable

##### List of audit links if any:

https://skynet.certik.com/projects/immutable-x

##### Website Link:

https://immutable.com/

##### Current TVL:

$27.79 M

##### Chain:

Ethereum

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)

https://www.coingecko.com/en/coins/immutablex

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)

https://coinmarketcap.com/currencies/immutable-x/

##### Short Description (to be shown on DefiLlama):

Immutable is the leading platform for building web3 games on Ethereum. Powering the next generation of web3 games.

##### Token address and ticker if any:

https://etherscan.io/token/0xf57e7e7c23978c3caec3c3548e3d615c346e79ff

##### Category (full list at https://defillama.com/categories) *Please choose only one:

NFT Marketplace

##### methodology (what is being counted as tvl, how is tvl being calculated):

Collateral deposited into the Immutable bridge to trade NFTs

##### Github org/user (Optional, if your code is open source, we can track activity):

https://github.com/immutable/

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/dydx exports TVL: 
        
 
 
    ethereum                  350.79 M
    
    total                    350.79 M

**Comment by llamatester** [comment]
The adapter at projects/immutablex exports TVL: 
        
 
 
    ethereum                  27.72 M
    
    total                    27.72 M

File: projects/immutablex/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const ADDRESSES = require('../helper/coreAssets.json');
+const { sumTokensExport } = require('../helper/sumTokens');
+
+const owners = [
+  '0x5FDCCA53617f4d2b9134B29090C87D01058e27e9',
+];
+
+const tokens = [
+  ADDRESSES.null,
+  ADDRESSES.ethereum.OMI,
```
Comment (g1nt0ki) [inline projects/immutablex/index.js:10]: arent these two their own tokens?

File: projects/helper/coreAssets.json [hunk 2]
```diff
@@ -37,6 +37,8 @@
     "LUSD": "0x5f98805A4E8be255a32880FDeC7F6728C6568bA0",
     "sUSD_OLD": "0x57ab1e02fee23774580c119740129eac7081e9d3",
     "sUSD": "0x57Ab1ec28D129707052df4dF418D58a2D46d5f51",
+    "OMI": "0xeD35af169aF46a02eE13b9d79Eb57d6D68C1749e",
```
Comment (g1nt0ki) [inline projects/helper/coreAssets.json:40]: for now, I am trying to keep this list short, main reason is, this list sort of acts as base Tokens while pricing some unknown token

**Review by g1nt0ki — COMMENTED** [review]
.

File: projects/immutablex/index.js (hunk 1 above)
Comment (0xmichalis) [reply projects/immutablex/index.js:10]: I don't think any of these are their own tokens. OMI is the currency of [Ecomi](https://www.ecomi.com/) which is a mobile platform that has [partnered with Immutable](https://medium.com/ecomi/veve-moves-to-ethereum-via-immutable-x-d69bfbd736d6) and GODS is the currency of Gods Unchained who [have launched on Immutable](https://blog.godsunchained.com/2021/03/04/immutable-x-gu-rollout/).

**Comment by llamatester** [comment]
The adapter at projects/dydx exports TVL: 
        
 
 
    ethereum                  349.36 M
    
    total                    349.36 M

**Comment by llamatester** [comment]
The adapter at projects/immutablex exports TVL: 
        
 
 
    ethereum                  28.23 M
    
    total                    28.23 M

**Pull request merged**